web/src/main/webapp/xslt/common/render-html.xsl

```diff
@@ -46,7 +46,7 @@
                select="'dataset'"/>
 
 
-    <html>
+    <html ng-app="{$angularModule}" lang="{$lang2chars}" id="ng-app">
```

Comment:
Even if using the formatter display app which allows to enable JS & angular in a formatter (eg. http://localhost:8080/geonetwork/srv/fre/display#78f93047-74f8-4419-ac3d-fc62e4b0477b/formatters/xsl-view) it does not bootstrap the angular app twice. So it looks fine.